app/timetables/admin.py

```diff
@@ -1,7 +1,7 @@
 from django.contrib import admin
 
 from .models import (Weekday, MealOption, Course, Meal, Timetable,
```

Comment:
@andela-kanyanwu Check [this](http://programmers.stackexchange.com/questions/187403/import-module-vs-from-module-import-function) and make a decision on which you would prefer